softdev.bib

```diff
@@ -2008,3 +2008,55 @@ @article{turner77error
     month = Aug,
     doi = "10.1016/0020-0190(77)90023-0"
 }
+
+@article{mccarthy60recursive,
+    author = "McCarthy, John",
+    title = "Recursive Functions of Symbolic Expressions and Their Computation by Machine, Part I",
+    journal = "Commun. ACM",
```

Comment:
I think we call this "CACM" elsewhere?

Reply:
It's Commun.~ACM elsewhere, I've added the tilde.